tests/config/checker/checkers/test_checker.py

```diff
@@ -53,13 +58,17 @@ def test_check(self):
             Config.check()
             assert len(Config._collector.errors) == 0
 
-        with mock.patch.object(logging.Logger, 'error') as error_message:
-            with mock.patch.dict(os.environ, {"FOO": "foo"}):
-                Config.global_config.clean_entities_enabled = "ENV[FOO]"
-                Config._collector = IssueCollector()
-                with pytest.raises(ConfigurationIssueError):
-                    Config.check()
-                assert len(Config._collector.errors) == 1
+        with mock.patch.dict(os.environ, {"FOO": "foo"}):
+            Config.global_config.clean_entities_enabled = "ENV[FOO]"
+            Config._collector = IssueCollector()
+            with pytest.raises(SystemExit):
+                Config.check()
+            expected_error_message = (
+                "`clean_entities_enabled` field of GlobalAppConfig must be populated with a"
+                ' boolean value. Current value of property `clean_entities_enabled` is "ENV[FOO]".'
```

Comment:
This is a wrong message. the current value of property `clean_entities_enabled` is "foo"and not "ENV[FOO]".
In any case, I believe we should just display the message. Eventually we can improve the message if it is not clear enough.

Reply:
I think `"ENV[FOO]"` is clear enough, no? It indicates that the user needs to check the environment variable again.